test/runtests.jl

```diff
@@ -48,8 +48,10 @@ using MixedLayerThermoclineDynamics, OffsetArrays, Test
 end
 
 @time @testset "Field tests" begin
-    nx, ny = 10, 12
-    Lx, Ly = 2.0, 2.4
+
+    include("test_fields.jl")
+    nx, ny = 20, 24
```

Comment:
```suggestion
    nx, ny = 100, 124
```
The tolerance was too low.

We had figured this out in #7. That's why it's better not to close the PRs but modify them... sometimes some things get lost. :)

Perhaps you can also make the test cleaner in the spirit of how they were done in #7?

Reply:
Yes, I can do that.

Reply:
Did you? Shall I?